docs/src/get-started/tutorials/get-started.py

```diff
@@ -0,0 +1,172 @@
+    max_radial=9,
+    max_angular=6,
+    atomic_gaussian_width=0.3,
+    center_atom_weight=1.0,
+    radial_basis={"Gto": {"spline_accuracy": 1e-6}},
+    cutoff_function={"ShiftedCosine": {"width": 0.5}},
+    radial_scaling={"Willatt2018": {"scale": 2.0, "rate": 1.0, "exponent": 4}},
+)
+
+# %%
+# After we set the hyper parameters we initilize a calculator object linking the the hyper parameters
+# with the a descriptor method.
+
+calculator = SphericalExpansion(**HYPER_PARAMETERS)
+descriptor0 = calculator.compute(frame0)
+print(type(descriptor0))
+
+# %%
+# The descriptor format is a :class:`equistore.tensor.TensorMap` object.
+# Equistore can be like Numpy for storing representation of atomistic ML data.
+# Extensive details on the equistore are be covered in the docs of
+# `equistore <https://lab-cosmo.github.io/equistore/>`_.
+# We now access the actual descriptor data atrributed to the
+# :class:`equistore.tensor.TensorMap` object.
+
+
+print(descriptor0)
+
+# %%
+#
+# The :class:`equistore.tensor.TensorMap` is structured in so called
+# ``blocks``. One block for each angular channel
+# and a combination of two elements labeled by their atomic number, e.g. 1 for
+# hydrogen, 6 for carbon etc.
+# Our descriptor contains 112 blocks covering
+# all combinations of the angular channels and the the elements.
+#
+# Lets take a look at the first block in details. The 0th block contains the
+# descriptor for the 0th angular channel between the hydrogen-hydrogen pairs.
+# The values are stored in an array where the first dimension is 8, because we have 8
+# hydrogen atoms in our frame. The last dimensions represents the number
+# of radial channels. We choose 9 above so there are 9 values in the last dimensions.
+
+print(descriptor0.block(0).values.shape)
+
+# %%
+# Lets explicitly take a look the values also called features of the first hydrogen.
+
+print(descriptor0.block(0).values[0])
+
+# %%
+# As you see the values are floating point numbers between -1 and 1.
+# Values in this range are reasonable and
+# can be directly used as an input of a machine learning algorithm.
+#
+# Rascaline is also able to process more than one structure within one function
+# call. You can process a whole dataset with
+
+descriptor_full = calculator.compute(frames)
+print(descriptor_full.block(0).values.shape)
+
+# %%
+# Now, the 0th block of the :class:`equistore.tensor.TensorMap` contains not 8
+# but 420 entries in the first dimensions.
+# This reflects the fact that in total we have 420 H–H pairs in the whole dataset.
+#
+#
+# If you want to use another calculator as the ``SphericalExpansion`` used here
+# check out the :ref:`userdoc-references` section. Details how to
+# calculate the features using another
+# programming language like Rust or C/C++ are given in the :ref:`userdoc-how-to`.
+#
+# Computing gradients
+# -------------------
+#
+# Additionally, rascaline is also able to calculate the gradients with respect to positions, which is
```

Comment:
should we mention gradients w.r.t. cell here as well?

Reply:
I thought about is as well. I maybe makes sense to mention it. I will write a sentence.